src/components/MainComponent/Description/Description.tsx

```diff
@@ -29,11 +28,11 @@ const Description = () => {
     }, [buttonName]);
 
     const handlePagination = () => {
-        const isMoreThanSix = meals.length - mealsForShow.length >= 6;
-        const offset = meals.length - mealsForShow.length;
+        const isMoreThanSix: boolean = meals.length - mealsForShow.length >= 6;
+        const offset: number = meals.length - mealsForShow.length;
         if (offset >= 1) {
-            const mealsForShowLength = mealsForShow.length;
-            let newMealsForShow;
+            const mealsForShowLength: number = mealsForShow.length;
```

Comment:
You don't need to define type number here Typescript will return this type by default